### PR TITLE
CVE-2014-0095

### PR DESCRIPTION
**Pull request by TheGuywithTheHat**
Filled out CVE-2014-0095

### CODE REVIEW

File: cves/CVE-2014-0095.yml [hunk 1]
```diff
@@ -204,4 +214,21 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    In my opinion, this vulnerability is not inherently a security bug. By
```
Comment (Anorwen) [inline cves/CVE-2014-0095.yml]: Is it impossible for this bug to be exploited and cause a full blow denial of service attack? Are you stating here that a user can not interact with the system in a way to make use of this bug and force threads to hang?

File: cves/CVE-2014-0095.yml [hunk 2]
```diff
@@ -101,9 +103,12 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: |
+    The original code was unit tested, but they did not catch the
+    vulnerability. The commit that fixed the vulnerability also added unit
```
Comment (Anorwen) [inline cves/CVE-2014-0095.yml]:
```suggestion
    vulnerability. The commit that fixed the vulnerability also added new
```

File: cves/CVE-2014-0095.yml [hunk 3]
```diff
@@ -101,9 +103,12 @@ unit_tested:
...
+    tests to check for it.
```
Comment (Anorwen) [inline cves/CVE-2014-0095.yml]:
```suggestion
    unit tests to verify that the vulnerability was resolved.
```

File: cves/CVE-2014-0095.yml [hunk 4]
```diff
@@ -130,8 +137,8 @@ subsystem:
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
     Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
-  name:
+  answer: The AJP request processor
+  name: ajp
```
Comment (Anorwen) [inline cves/CVE-2014-0095.yml:135]:
```suggestion
  name: AJP
```

Comment (TheGuywithTheHat) [reply cves/CVE-2014-0095.yml:135]: Based on the examples given I assume this is supposed to be lowercase.

File: cves/CVE-2014-0095.yml (hunk 1 above)
Comment (TheGuywithTheHat) [reply cves/CVE-2014-0095.yml]: I agree that I was not very clear there, so I tried to clean it up a bit. Do you think my changes in commit cf508086f6399770bbd4a18b9156f5bcc52f6936 are enough?

Comment (Anorwen) [reply cves/CVE-2014-0095.yml]: That looks better. The one thing I do see is that it looks like you miss typed "quite" as "qute". Other than that it looks good.

**Comment by Anorwen** [comment]
@TheGuywithTheHat I would give this 2 points because it is a difficult vulnerability to find.

**Pull request merged**